bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -152,63 +157,70 @@ public static String executeUrl(String httpMethod, String url, Properties httpHe
 
         HttpClient client = new HttpClient();
```

Comment:
Correct, normally it should be enough to create one client in constructor, start it on initialize and stop it on dispose.